Modules/Scripted/DICOMLib/DICOMUtils.py

```diff
@@ -156,3 +156,180 @@ def __enter__(self):
     return slicer.dicomDatabase
   def __exit__(self, type, value, traceback):
     closeTemporaryDatabase(self.originalDatabaseDir)
+
+#------------------------------------------------------------------------------
+# Download file to local storage.
+# Target file path needs to indicate the file name and extension as well
+def downloadFile(url, targetFilePath):
```

Comment:
Consider adding the description of the method as a python docstring:

```
def downloadFile(url, targetFilePath):
  """Download ``url`` to local storage as ``targetFilePath``.
  """
```

If ``targetFilePath`` extension is required, we could check for it using https://docs.python.org/2/library/os.path.html#os.path.splitext and return an exception. 

If this function is general enough, may it could be added to ``slicer.util`` instead.